pandas/core/reshape/concat.py

```diff
@@ -145,13 +148,22 @@ def concat(
     copy : bool, default True
         If False, do not copy data unnecessarily.
 
+    initializer : a Series or DataFrame object, default None
```

Comment:
It seems that there is an extra newline between ``copy`` and ``initializer`` docstrings.